core/src/verify.rs

```diff
@@ -454,6 +487,22 @@ impl CommitSequenceVerifier {
         self.total_commits.push(commit.clone());
         Ok(())
     }
+
+    /// To verify that the newly added previous_block_hash field effectively prevents replay attacks
+    /// Check the integrity of the chain of agendas when processing new agendas
+    pub fn verify_agenda_integrity(
+        new_agenda: &Agenda,
```

Comment:
```suggestion
        agenda: &Agenda,
```

Reply:
reflected

Reply:
Double check please, I cannot see the change.

Reply:
sorry for that. there was a mistake from my side.